tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Repository/OdsRepositoryTests.cs

```diff
@@ -115,7 +133,7 @@ public static async Task GetBuyerOrganisationByOdsCode_CallsOdsApi_Once()
         [SuppressMessage("ReSharper", "ObjectCreationAsStatement", Justification = "Testing")]
         public static void Constructor_IOdsRepository_OdsSettings_NullSettings_ThrowsException()
         {
-            Assert.Throws<ArgumentNullException>(() => new OdsRepository(null));
+            Assert.Throws<ArgumentNullException>(() => new OdsRepository(null, new LazyCache.CachingService()));
```

Comment:
Is `LazyCache.` redundant here? Can we simplify this to `new CachingService()`?

Reply:
done